features/step_definitions/applications/applications_steps.rb

```diff
@@ -9,6 +9,15 @@
   cinstance.save!
 end
 
+Given "{buyer} has application {string} with plan {string}" do |buyer, name, plan_name|
+  plan = ApplicationPlan.find_by!(name: plan_name)
+  assert buyer.provider_account.application_plans.include?(plan)
+  FactoryBot.create(:cinstance, :user_account => buyer,
+                    :plan         => plan,
+                    :name         => name)
```

Comment:
doesn't rubocop complain for using `:sym => value` instead of `sym: value`?

Reply:
No, are you using the `sym: value` syntax as a style rule in the project?

Reply:
I don't know if we use in rubocop but I think we always use it.

Reply:
done